storm-server/src/main/java/org/apache/storm/scheduler/TopologyDetails.java

```diff
@@ -193,22 +200,44 @@ private void initResourceList() {
     }
 
     /**
-     * Returns a representation of the non-system components of the topology graph Each Component object in the returning map is populated
+     * Returns a representation of the non-system components of the topology graph. Each Component object in the returning map is populated
      * with the list of its parents, children and execs assigned to that component.
      *
      * @return a map of components
      */
-    public Map<String, Component> getComponents() {
+    public Map<String, Component> getTopoComponents() {
```

Comment:
Can this method retain the old name? And the getAllComponents be an overloaded getComponents()?